generators/java/sdk/src/main/java/com/fern/java/client/generators/RootClientGenerator.java

```diff
@@ -201,6 +201,14 @@ private TypeSpec getClientBuilder() {
                     .build());
         }
 
+        clientBuilder.addMethod(MethodSpec.methodBuilder("timeout")
+            .addModifiers(Modifier.PUBLIC)
+            .addParameter(int.class, "timeout")
+            .returns(builderName)
+            .addStatement("this.$L.timeout(timeout)", CLIENT_OPTIONS_BUILDER_NAME)
+            .addStatement("return this")
+            .build());
```

Comment:
nit: can we add a docstring to this method that this sets the timeout in seconds

Reply:
Done